code/__DEFINES/machines.dm

```diff
@@ -97,7 +97,8 @@
 #define SUPERMATTER_WARNING 3		// Ambient temp > CRITICAL_TEMPERATURE OR integrity damaged
 #define SUPERMATTER_DANGER 4		// Integrity < 50%
 #define SUPERMATTER_EMERGENCY 5		// Integrity < 25%
-#define SUPERMATTER_DELAMINATING 6	// Pretty obvious.
+#define SUPERMATTER_DESTABILIZED 6  // Crystal Matrix destabilized
+#define SUPERMATTER_DELAMINATING 7	// Pretty obvious.

```

Comment:
[add dmdoc to meet documentation requirements](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
right i'll do it next commit